account_move_update_analytic/views/account_move_view.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Le Filament
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="account_move_form_view">
+        <field name="name">account.move.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='invoice_line_ids']/tree/field[@name='analytic_tag_ids']"
+                position="after"
+            >
+                <button
+                    name="account_move_update_analytic.action_view_account_move_update_analytic"
+                    string="Update Analytic"
+                    type="action"
+                    groups="analytic.group_analytic_accounting,analytic.group_analytic_tags"
+                    attrs="{'invisible': [('parent.state', '!=', 'posted'),]}"
+                />
+            </xpath>
+            <xpath
+                expr="//field[@name='line_ids']/tree/field[@name='analytic_tag_ids']"
+                position="after"
+            >
+                <button
+                    name="account_move_update_analytic.action_view_account_move_update_analytic"
+                    string="Update Analytic"
+                    type="action"
+                    groups="analytic.group_analytic_accounting,analytic.group_analytic_tags"
+                    attrs="{'invisible': [('parent.state', '!=', 'posted'), ('display_type', 'in', ('line_section', 'line_note'))]}"
```

Comment:
Use `parent_state` instead of `parent.state`. It's already computed stored=True.
```suggestion
                    attrs="{'invisible': [('parent_state', '!=', 'posted'), ('display_type', 'in', ('line_section', 'line_note'))]}"
```

Reply:
I then needed to add this parent_state field as invisible before in view, but done